docx/oxml/__init__.py

```diff
@@ -190,6 +192,8 @@ def OxmlElement(nsptag_str, attrs=None, nsdecls=None):
 register_element_cls('w:pPr',             CT_PPr)
 register_element_cls('w:pStyle',          CT_String)
 register_element_cls('w:spacing',         CT_Spacing)
+register_element_cls('w:tab',             CT_TabStop)
+register_element_cls('w:tabs',            CT_TabStops)
 register_element_cls('w:widowControl',    CT_OnOff)
```

Comment:
@DKWoods All these changes are merged in here: https://github.com/python-openxml/python-docx/commits/feature/tabstops

..these comments are just so you can know for next time. Check the merged commit for actual specifics.

* w:tab/CT_TabStop is not needed to pass this test. It will have to wait it's turn :) Dropped from this commit.